WebApp-OpenIDConnect-DotNet/Views/Shared/_LoginPartial.cshtml

```diff
@@ -2,6 +2,8 @@
 {
     <ul class="nav navbar-nav navbar-right">
         <li class="navbar-text">Hello @User.Identity.Name!</li>
+        <li><a asp-area="" asp-controller="Session" asp-action="EditProfile">Edit Profile</a></li>
+        <li><a asp-area="" asp-controller="Session" asp-action="ResetPassword">Reset Password</a></li>
```

Comment:
We have built password change into our profile edit screens directly. Password reset is then only for an unauthenticated "forgot my password" scenario.